src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php

```diff
@@ -133,14 +134,4 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             ));
         });
     }
-
-    /**
-     * {@inheritdoc}
-     *
-     * Add the Sulu environment to the container name
-     */
-    protected function getContainerClass()
```

Comment:
Don't we need that function at all anymore? Should also other occurences be removed?

Reply:
This seems something which I did forget to be update in https://github.com/sulu/sulu/pull/5184. I will great an own PR for this as this need to be also changed in 2.0

Reply:
Own PR for 2.0 branch created here: https://github.com/sulu/sulu/pull/5326

Reply:
I have merge #5326, and merged it back to release/2.1. Can you also rebase that branch now?